tools/retrieve_prs.py

```diff
@@ -20,21 +24,16 @@
 )
 
 
-def run(command):
-    """Returns (return-code, stdout, stderr)"""
-    p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
-    output, err = p.communicate()
-    rc = p.returncode
-    enc = locale.getpreferredencoding()
-    output = output.decode(enc)
-    err = err.decode(enc)
-    return rc, output.strip(), err.strip()
+def _run_cmd(cmd):
+    try:
+        return subprocess.check_output(cmd, cwd=ROOT_DIR).decode('ascii').strip()
+    except Exception:
+        return None
 
 
 def commit_title(commit_hash):
-    cmd = f"git log -n 1 --pretty=format:%s {commit_hash}"
-    ret, out, err = run(cmd)
-    return out if ret == 0 else None
+    cmd = ['git', 'log', '-n', '1', f'--pretty=format:%s {commit_hash}']
```

Comment:
I think this is different from what it used to be.
```suggestion
    cmd = ['git', 'log', '-n', '1', '--pretty=format:%s', f'{commit_hash}']
```